### ISSUE TITLE
Split tests in multiple files

### ISSUE THREAD

**Issue by pBauma** [mmurdoch/arduinounit#67]
Is there a way to split the tests in multiple files and have one central point to start them from? 
It would make my testing code much cleaner.

**Comment by wmacevoy** [comment]
Yes, just declare them in multiple files.  Test::run() will run them all.
